x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_add_edit/form/defaults.tsx

```diff
@@ -4,8 +4,15 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { formatKibanaNamespace } from '../../../../../../common/formatters';
+import { INVALID_NAMESPACE_CHARACTERS } from '@kbn/fleet-plugin/common';
 import { DEFAULT_FIELDS } from '../constants';
+
+/* Formats kibana space id into a valid Fleet-compliant data-stream namespace */
+export const formatKibanaNamespace = (spaceId: string) => {
```

Comment:
This is repeated. I believe this already exists now in `private_formatters`

Ultimately though, this is relevant for both cloud and private locations.

Can we move this to somewhere common?

It's also imported in `add_monitor_project` but I don't believe the import has been adjusted.